page-objects/src/components/sidebar/WelcomeContent.ts

```diff
@@ -0,0 +1,40 @@
+import { By, WebElement } from "selenium-webdriver";
+import { ViewSection } from "./ViewSection";
+import { AbstractElement } from "../AbstractElement";
+
+export class WelcomeContentButton extends AbstractElement {
+  constructor(
+    private panel: WebElement,
+    public welcomeSection: WelcomeContentSection
+  ) {
+    super(panel, welcomeSection);
+  }
+
+  public async getTitle(): Promise<string> {
+    return this.panel.getText();
+  }
+
+  public async takeAction(): Promise<void> {
+    await this.click();
+  }
+}
+
+export class WelcomeContentSection extends AbstractElement {
+  constructor(public parent: ViewSection, panel: WebElement) {
+    super(panel);
```

Comment:
For the sake of consistence, please use the parent element as the second argument. Also, looks like you forgot to feed the parent into the super constructor.